packages/actions/docs/04-modals.md

```diff
@@ -323,17 +323,19 @@ Action::make('updateAuthor')
 
 ## Changing the modal width
 
-You can change the width of the modal by using the `modalWidth()` method. Options correspond to [Tailwind's max-width scale](https://tailwindcss.com/docs/max-width). The options are `xs`, `sm`, `md`, `lg`, `xl`, `2xl`, `3xl`, `4xl`, `5xl`, `6xl`, `7xl`, and `screen`:
+You can change the width of the modal by using the `modalWidth()` method. Options correspond to [Tailwind's max-width scale](https://tailwindcss.com/docs/max-width). The options are `ExtraSmall`, `Small`, `Medium`, `Large`, `ExtraLarge`, `TwoExtraLarge`, `ThreeExtraLarge`, `FourExtraLarge`, `FiveExtraLarge`, `SixExtraLarge`, `SevenExtraLarge`, and `Screen`:
```

Comment:
@danharrin these are inconsistent with `IconEntrySize::ExtraExtraLarge` and `IconColumnSize::ExtraExtraLarge` that @binaryfire recently introduced.

Reply:
Which I renamed 😉

Reply:
at one point last night during my refactor, that was on my screen